src/Uno.UI/UI/Xaml/Internal/InputManager.Pointers.Managed.cs

```diff
@@ -352,6 +352,12 @@ private void OnPointerExited(Windows.UI.Core.PointerEventArgs args)
 
 		private void OnPointerPressed(Windows.UI.Core.PointerEventArgs args)
 		{
+			// If 2+ mouse buttons are pressed, we only respond to the first.
+			if (args.CurrentPoint.PointerDeviceType == PointerDeviceType.Mouse && _pressedElements.Count != 0)
```

Comment:
I'm not sure to understand what kind of bug we are trying to fix here. I suspect this will cause more issue than it will solve. The _pressedElememts is already a "last chance" fix, I would try to not build new feature on it.

Reply:
Yeah, I forgot that we discussed that _pressedElements should be removed at some point.
The bug is https://github.com/unoplatform/uno/issues/16801 (see also the runtime test in this PR). Basically, clicking multiple mouse buttons shouldn't really do anything (on WinUI). On our side, not only does each button press fire an additional PointerPressed, but it also leads to really weird behaviour. E.g. if you left press on something, move the pointer, and right click on something else, that something else will see that the left button is pressed, so it will handle it as a left press.